configs/yolo/yolov3_d53_320_273e_coco_ipu.py

```diff
@@ -0,0 +1,98 @@
+_base_ = './yolov3_d53_mstrain-608_273e_coco.py'
+# dataset settings
+IM_SIZE = 320
+custom_hooks = [dict(type='NumClassCheckHook'), dict(type='BNToFP32')]
+img_norm_cfg = dict(mean=[0, 0, 0], std=[255., 255., 255.], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='LoadAnnotations', with_bbox=True),
+    dict(
+        type='Expand',
+        mean=img_norm_cfg['mean'],
+        to_rgb=img_norm_cfg['to_rgb'],
+        ratio_range=(1, 2)),
+    dict(
+        type='MinIoURandomCrop',
+        min_ious=(0.4, 0.5, 0.6, 0.7, 0.8, 0.9),
+        min_crop_size=0.3),
+    dict(type='Resize', img_scale=(IM_SIZE, IM_SIZE), keep_ratio=True),
+    dict(type='RandomFlip', flip_ratio=0.5),
```

Comment:
Don't need `PhotoMetricDistortion` and `Normalize`?

Reply:
In order to save memory, we use int8 input, so normalize is moved into ipu. If PhotoMetricDistortion is added and then quantized image data to int8, there will be a loss of information, so it is also canceled. Fortunately, train can  SOTA without PhotoMetricDistortion

Reply:
If so, please add a comment to the config explaining why `PhotoMetricDistortion` is not used.